src/library/itunes/itunesschema.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+#include <QString>
+
+#define ITUNES_LIBRARY_TABLE "itunes_library"
+#define ITUNES_PLAYLISTS_TABLE "itunes_playlists"
+#define ITUNES_PLAYLIST_TRACKS_TABLE "itunes_playlist_tracks"
```

Comment:
Implementation note: I've also extracted the names of the iTunes tables to avoid hardcoding them in different places (though admittedly not the column names since using those as constants would get excessively verbose e.g. in `ITunesDAO` and drastically reduce readability of the admittedly already quite long queries).

Reply:
can you make them `constexpr static`? or QStrings? Or is there a good reason for choosing a preprocessor define here?

Reply:
No particular reason aside from consistency with the other tables (e.g. as listed in `crateschema.h`)